docs/developers/README.md

```diff
@@ -143,16 +143,21 @@ with the binary and file(s) is mounted.
 If the image is a private registry, the service account should include an
 [ImagePullSecret](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account)
 
-## Builder namespace on containers
+## Reserved directories
 
 The `/tekton/` directory is reserved on containers for internal usage. Examples
 of how this directory is used:
 
-- Task Results are written to `/tekton/results`
-- Various tools like the entrypoint are placed in `/tekton/tools`
-- The termination log message is written to `/tekton/termination`
-- Sequencing step containers is done using both `/tekton/downward/ready`
-and numbered files in `/tekton/tools`
+* `/workspace` - This directory is where [resources](#resources) and [workspaces](#workspaces)
+  are mounted.
+* `/tekton` - This directory is used for Tekton specific functionality:
+  * These folders are [part of the Tekton API](../api_compatibility_policy.md):
+    * `/tekton/results` is where [results](#results) are written to
+  * These folders are [implementation details of Tekton](developers/README.md):
```

Comment:
"...and users should not rely on this specific behavior, as it may change in the future."

Reply:
We should document variables for those path — and if there is not, we should add some :upside_down_face:

Reply:
Good thinking. `$(results.foo.path)` was added so that users don't have to hard-code `/tekton/results/foo`.